cardinal/world.go

```diff
@@ -16,7 +17,10 @@ import (
 )
 
 type World struct {
-	impl            *ecs.World
+	implWorld       *ecs.World
```

Comment:
i think i prefer just having `impl`, as now we have a bit of a stuttering problem here e.g.

```go
// stutter
server := http.HTTPServer{}

// no stutter
server := http.Server{}

```

for example, if an object that contains world as field names that field `world`, we then get `world.implWorld`, which is kinda redundant imo. not a big deal in the grand scheme, but maybe this is a chance to get a consistent codebase policy around stuttering. thoughts? cc @jerargus @namrapatel @pyrofolium

Reply:
impl is a bigger problem imo, because what is it? Is it implementing server? Is is implementing world? Is it implementing gameManager? 

Also impl itself is a bad name even without the additional because it's an acronym that can get mistaken for some other meaning. My philosophy has been clarity over aesthetics.

If your want to get rid of the "stuttering" we can change the name, my main issue with impl is that it's not clear at all.